src/Statistics/Histogram.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "Histogram.h"
```

Comment:
You have one .cpp file for two header files. The .cpp file should be split (and one of the two headers renamed appropriately). E.g.: `HistrogramBuilder.cpp/h`.